packages/react-native/src/evm/components/ConnectWalletFlow/SmartWallet/SmartWalletFlow.tsx

```diff
@@ -37,6 +37,7 @@ export const SmartWalletFlow = ({
   const walletObj = useSupportedWallet(walletIds.smartWallet) as SmartWalletObj;
   const connect = useConnect();
   const targetChain = useWalletContext().activeChain;
+  const colorScheme = useColorScheme();
```

Comment:
Should this be `const theme = useTheme()` instead of `useColorScheme`?

Reply:
theme returns the entire theme object, we only need 'light' | 'dark' here